graphrag/api/index.py

```diff
@@ -27,6 +28,7 @@ async def build_index(
     memory_profile: bool = False,
     callbacks: list[WorkflowCallbacks] | None = None,
     progress_logger: ProgressLogger | None = None,
+    use_new_pipeline: bool = False,
```

Comment:
This is for backwards compat of for the draft purpose?

Reply:
just for draft so you can run either way